README.md

```diff
@@ -123,6 +123,14 @@ Note that if you are using ``@edx/frontend-platform``'s ``AppProvider`` componen
 
 ## Contributing
 
+The branch to target with your PR depends on the type of change you are contributing to Paragon.
```

Comment:
Should we also the remove [previously added section](https://github.com/openedx/paragon?tab=readme-ov-file#opening-pull-requests-for-breaking-changes) in this PR?

Reply:
good call, removed in https://github.com/openedx/paragon/pull/3344/commits/e11e7cf043f58e9fed5ffe1d603b5c1287b45f15